vector/src/main/java/im/vector/app/features/analytics/DecryptionFailureTracker.kt

```diff
@@ -40,8 +40,8 @@ private data class DecryptionFailure(
 )
 private typealias DetailedErrorName = Pair<String, Error.Name>
 
-private const val GRACE_PERIOD_MILLIS = 4_000
-private const val CHECK_INTERVAL = 2_000L
+private const val GRACE_PERIOD_MILLIS = 30_000
+private const val CHECK_INTERVAL = 40_000L
```

Comment:
I quickly check the code to see that the change from `GRACE_PERIOD_MILLIS > CHECK_INTERVAL` to `GRACE_PERIOD_MILLIS < CHECK_INTERVAL` was OK, and I think it's fine this those 2 value are unrelated.

Reply:
Actually that's right. Should probably be a bit lower.
Because if currently the error happens just after a `checkFailures` (called on `CHECK_INTERVAL` tick`), then that error will be checked after 40s. So it will mean that the grace period is extended to 40s?
I'll put 10s so that it's better respecting the grace period. Worst case could extend to 39s

Reply:
I need some additional work, this code is not working as I thought. It's gracing events if you close the timeline, and this won't work properly with an extended grace period